includes/Hooks.php

```diff
@@ -285,7 +285,11 @@ public function onHtmlPageLinkRendererBegin( $linkRenderer, $target, &$text,
 			$customAttribs['class'] = '';
 		}
 
-		$customAttribs['class'] .= 'fw-link';
+		$config = MediaWikiServices::getInstance()->getService( 'ConfigFactory' )
+			->makeConfig( 'femiwiki' );
```

Comment:
Because this function uses HookContainer system, I think it is possible to inject ConfigFactory as a service.

Reply:
Resolving this because it is not urgent.

Reply:
Sorry! Did see this. Just been occupied with other things. Should I open an issue to track this?

Reply:
https://github.com/femiwiki/FemiwikiSkin/issues/483 is for that and I thought fixing the failed test is more important, so it's completely okay!